### PR TITLE
Contact page new UI

### PR DESCRIPTION
**Pull request by johnnyd-eth**
Implements: https://www.figma.com/file/UE4g6PFX3d7odydhzAeD5j/JBM-Website-Updates-March-2023?node-id=1694-189034&t=VqqFo1vN7oOAC0lL-0

Fixes JB-322

Screen Shot 2023-05-02 at 3 38 11 pm

Screen Shot 2023-05-02 at 3 38 18 pm

### CODE REVIEW

File: public/assets/images/contact-hero-od.png
Comment (tomquirk) [inline public/assets/images/contact-hero-od.png:1]: webp https://discord.com/channels/939317843059679252/1040815900615507988/1102451736372838461

File: public/assets/images/contact-hero-ol.png
Comment (tomquirk) [inline public/assets/images/contact-hero-ol.png:1]: webp https://discord.com/channels/939317843059679252/1040815900615507988/1102451736372838461

File: src/pages/contact/Contact.tsx [hunk 1]
```diff
@@ -126,23 +133,32 @@ export default function Contact() {
...
-                <Trans>Send Message</Trans>
-              </Button>
+              <div>
+                <Button
+                  size="large"
+                  type="primary"
+                  htmlType="submit"
+                  loading={loading}
+                >
+                  <Trans>Send Message</Trans>
```
Comment (tomquirk) [inline src/pages/contact/Contact.tsx:159]: wrap in span so the loading icon doesnt get fucked

File: src/pages/contact/Contact.tsx [hunk 2]
```diff
@@ -159,15 +175,18 @@ export default function Contact() {
...
+          <Col xs={24} md={11} className="hidden md:flex">
+            <Image
+              src={
+                forThemeOption?.({
+                  light: contactHeroLight,
+                  dark: contactHeroDark,
+                }) ?? ''
+              }
+              alt="Banny making a phone call"
+              loading="lazy"
```
Comment (tomquirk) [inline src/pages/contact/Contact.tsx:187]: dont lazy load, cos the image will be above the fold most of the time

**Review by tomquirk — APPROVED** [review]
LGTM @johnnyd-eth, just check in on those imgs pls

File: public/assets/images/contact-hero-od.png
Comment (johnnyd-eth) [reply public/assets/images/contact-hero-od.png:1]: Ty sir, missed this

**Pull request merged**